### ISSUE TITLE
[MaterialContainerTransform] IllegalArgumentException: The transition must support seeking using fragment:1.7.0-beta01

### ISSUE THREAD

**Issue by Pittvandewitt** [material-components/material-components-android#4112]
**Description:** IllegalArgumentException when using MaterialContainerTransform in any project compiled with [fragment:1.7.0-beta01](https://developer.android.com/jetpack/androidx/releases/fragment#1.7.0-beta01).

```
FATAL EXCEPTION: main
Process: io.material.catalog, PID: 11196
java.lang.IllegalArgumentException: The Transition must support seeking.
	at androidx.transition.TransitionManager.controlDelayedTransition(TransitionManager.java:512)
	at androidx.transition.FragmentTransitionSupport.controlDelayedTransition(FragmentTransitionSupport.java:249)
	at androidx.fragment.app.DefaultSpecialEffectsController$TransitionEffect$onStart$4.invoke(DefaultSpecialEffectsController.kt:769)
	at androidx.fragment.app.DefaultSpecialEffectsController$TransitionEffect$onStart$4.invoke(DefaultSpecialEffectsController.kt:767)
	at androidx.fragment.app.DefaultSpecialEffectsController$TransitionEffect.runTransition(DefaultSpecialEffectsController.kt:1044)
	at androidx.fragment.app.DefaultSpecialEffectsController$TransitionEffect.onStart(DefaultSpecialEffectsController.kt:767)
	at androidx.fragment.app.SpecialEffectsController$Effect.performStart(SpecialEffectsController.kt:817)
	at androidx.fragment.app.SpecialEffectsController.processStart(SpecialEffectsController.kt:426)
	at androidx.fragment.app.SpecialEffectsController.executePendingOperations(SpecialEffectsController.kt:284)
	at androidx.fragment.app.Fragment$4.run(Fragment.java:2924)
	at android.os.Handler.handleCallback(Handler.java:959)
	at android.os.Handler.dispatchMessage(Handler.java:100)
	at android.os.Looper.loopOnce(Looper.java:232)
	at android.os.Looper.loop(Looper.java:317)
	at android.app.ActivityThread.main(ActivityThread.java:8501)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:552)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:878)
```

**Source code**: 
MaterialContainerTransform not returning `true` in `isSeekingSupported`

**Android API version:** 34 and up

**Material Library version:** 1.12.0-beta01

**Comment by dsn5ft** [comment]
Thanks for the report, we are talking to the Fragment team about this issue and what the fix should be. Even though MaterialContainerTransform does not return true for isSeekingSupported, this should not cause a crash.

**Comment by dsn5ft** [comment]
@Pittvandewitt would you be able to provide a sample project (e.g. via a GitHub repo) that reproduces the crash?

**Comment by Pittvandewitt** [comment]
> @Pittvandewitt would you be able to provide a sample project (e.g. via a GitHub repo) that reproduces the crash?

https://github.com/Pittvandewitt/PBFragmentSharedElement

**Comment by dsn5ft** [comment]
Thanks @Pittvandewitt! This helped uncover a bug in the Fragment library. They have fixed it in https://android-review.googlesource.com/c/platform/frameworks/support/+/3017979 which should be available in the upcoming Fragment 1.7.0-rc02 release.
